Firestore/core/src/firebase/firestore/local/local_serializer.cc

```diff
@@ -155,6 +169,19 @@ void LocalSerializer::EncodeDocument(Writer* writer,
   // Ignore Document.create_time. (We don't use this in our on-disk protos.)
 }
 
+void LocalSerializer::EncodeNoDocument(Writer* writer,
+                                       const model::NoDocument& no_doc) const {
+  // Encode Document.name
+  writer->WriteTag({PB_WT_STRING, google_firestore_v1beta1_Document_name_tag});
+  writer->WriteString(rpc_serializer_.EncodeKey(no_doc.key()));
+
+  // Encode Document.read_time
```

Comment:
Nit: the actual tag is `NoDocument_read_time`, not `Document_read_time`.

Reply:
Fixed. Also fixed above where I used `google_firestore_v1beta1_Document_name_tag` instead of `firestore_client_NoDocument_name_tag` in the code itself. (Though both evaluate to 1, hence the passing tests.)